docs/model-monitoring/applications.md

```diff
@@ -112,3 +112,42 @@ It is recommended to specify the exact version of the `evidently` package for re
 ```
 
 See a full example in [Realtime monitoring and drift detection](../tutorials/05-model-monitoring.ipynb#evidently-app).
+
+## Scaling
+
+The model monitoring application can be scaled to handle increased workloads through two key dimensions:
+
+1. **Workers**: Individual processes within each Kubernetes pod
+2. **Replicas**: The number of Kubernetes pods deployed
+
+These parameters are configured in the `config.py` file under the `application_stream_args` section:
+
+```py
+{
+    "application_stream_args": {
+        "v3io": {
+            "shard_count": 4,  # Number of V3IO shards
+            "retention_period_hours": 24,  # Data retention period
+            "num_workers": 4,  # Processes per pod
+            "min_replicas": 1,  # Minimum number of pods
+            "max_replicas": 1,  # Maximum number of pods
+        },
+        "kafka": {
+            "partition_count": 4,  # Number of Kafka partitions
+            "replication_factor": 1,  # Kafka replication factor
+            "num_workers": 4,  # Processes per pod
+            "min_replicas": 1,  # Minimum number of pods
+            "max_replicas": 1,  # Maximum number of pods
+        },
+    }
+}
+```
+
+### Scaling Rule
+
+The fundamental scaling rule to maintain is:
+
+**Shards/Partitions ≥ Replicas × Workers**
+
+In other words, the number of shards (V3IO) or partitions (Kafka) must be at least equal to the total number of worker processes across all pods.
+
```

Comment:
Any value in allowing Partitions != Replicas * Workers? If there isn't just mention they should equal.
Please also mention this constraint in the `shard_cont` / `partition_count` comments